package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "hapi": "^13.0.0",
+    "mongoose": "^4.4.4",
```

Comment:
This should be in dev-deps, not as deps because it is only run by the test. All users who need the mongoose model need to install by themselves.